src/providers/goDaddy.ts

```diff
@@ -1,56 +1,44 @@
-/*eslint @typescript-eslint/camelcase: 0*/
+/* eslint @typescript-eslint/camelcase: 0 */
+/* eslint @typescript-eslint/no-explicit-any: 0 */
+
 import { sendRequest } from '../helpers/httpRequest'
 import { getProviderKeys } from '../api/lib/data'
 import { Provider } from '../api/types/general'
+import { ServiceKey } from '../api/types/admin'
 
-export const getDomains = async (): Promise<Provider> => {
-  const service = 'https://api.godaddy.com'
-  const name = 'Godaddy'
-  const serviceKeys = getProviderKeys()
+const name = 'Godaddy'
+const service = 'https://api.godaddy.com'
```

Comment:
I wonder if this kind on `constant` would be a best practice in uppercase or not?

Reply:
yeah, uppercase is best.